docs/fast_data/faq/benchmark_fast_data.md

```diff
@@ -0,0 +1,155 @@
+---
+id: benchmark_fast_data
+title: Bootstrapping Fast Data
+sidebar_label: Bootstrapping Fast Data
+---
+
+This section provides guidelines and answers on how to provision the infrastructure components in order to have an up-and-running instance of Mia-Platform Fast Data. 
+
+In particular, it lists some of the minimum resources in terms of CPU or RAM that the components (Kafka, MongoDB and Mia-Platform Plugins) needs to satisfy, along with their configuration.
+
+:::info Before you start
```

Comment:
```suggestion
:::info 
Before you start
```

Reply:
hi @akelity 

i put after the info markdown so the panel will use it as title. with a line break the title is not recognized